modules/tasks/tasks/src/main/java/org/motechproject/tasks/service/impl/ChannelServiceImpl.java

```diff
@@ -177,6 +181,49 @@ protected void doInTransactionWithoutResult(TransactionStatus status) {
         return channels;
     }
 
+    private void registerLoggerChannel() {
+        ActionParameterRequest logMessage = new ActionParameterRequest(
+                0, //order
+                "message", // key
+                null, // default value
+                "task.logger.value", // display name
+                "TEXTAREA", // type (default: UNICODE)
+                true, // required
+                false, // hidden
+                null
+        );
+
+        TreeSet<ActionParameterRequest> myList = new TreeSet<ActionParameterRequest>();
+        myList.add(logMessage);
+
+        ActionEventRequest logAction = new ActionEventRequest(
+                "org.motechproject.tasks.service.ChannelService", // name
+                "task.logger.log", // display name
+                "LogMessage", // subject
+                null, // description
+                "org.motechproject.tasks.service.ChannelService", // service interface
+                "logMessage", // service method
+                "NAMED_PARAMETERS", // service method call manner (default: NAMED_PARAMETERS)
+                myList // action parameters
+        );
+
+        ChannelRequest logChannel = new ChannelRequest(
+                "task.logger", // display name
+                "org.motechproject.motech-tasks", // module name
+                "0.29.0.SNAPSHOT", // module version
+                null, // description
+                null, // trigger requests
+                Arrays.asList(logAction) // action requests
+        );
+
+        registerChannel(logChannel);
+    }
+
+    @Override
+    public void logMessage(String message) {
+        LOGGER.error(message);
```

Comment:
Btw. it would be probably nice to make the level configurable (via task action parameter)